src/models/fees-types.ts

```diff
@@ -0,0 +1,18 @@
+export enum Estimations {
+  Low = 0,
+  Middle = 1,
+  High = 2,
+  Custom = 3,
+}
```

Comment:
```suggestion
export enum Estimations {
  Low,
  Middle,
  High,
  Custom,
}
```

This change isn't necessary, as being explicit can help readability, but these are the default values when defining an enum